tests/replace.spec.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ReplacedBlock from 'b:ReplacedBlock';
+
+it('Entity should be replaced and save wrapper', () => {
+    const wrapper = shallow(<ReplacedBlock text="replacedWithThisText"/>).dive(),
```

Comment:
`wrap(wrpdblk) → wrap(props, state, component)` am I right that we breaking compatibility?
Is this a good API with 3 arguments instead of one? Why we can't get props/state from the instance (this.props/state)?

p.s. We really need a way to describe decls right here. We SHOULD NOT test babel-import plugin or webpack-bem-loader here to resolve `'b:*'` dependencies and it's better to put declarations right in test files (and remove fixtures folder) for readability purposes.

Reply:
C - Consistency!